docs/recursive-analysis.md

```diff
@@ -0,0 +1,91 @@
+# Recursive analysis
```

Comment:
We'll need to decide what to do with this.
The way it's currently outlined, it'll NOT show up in the left-side nav - and will therefore only be reachable either through direct links - or through the link in strategy-customization.

i'm ... tempted to combine this with "lookahead analysis" - as the commands are strongly related, and i think we already have a "too long" nav ...

Reply:
Personally, I felt like Backtesting, Hyperopt, Lookahead and Recursive analysis should all be combined into 1 main menu.

Reply:
you mean something like this

```
Backtesting
    Backtesting
    hyperopt
    lookahead analysis
    recursive analysis
```

not against this per se - though i don't actually like the behavior of expanding boxes, as it'd require 2 clicks to actually go to backtesting (for example).

Reply:
another place I have in mind is `Advanced Topics`

Reply:
but you'd keep it as 2 separate pages? or do something similar than what we do with plugins ... ?

Reply:
keep as separate pages.